app/upgrades/v19/upgrades.go

```diff
@@ -65,3 +76,28 @@ func MigrateRateLimitModule(ctx sdk.Context, k ratelimitkeeper.Keeper) {
 	hourEpoch.EpochStartHeight = ctx.BlockHeight()
 	k.SetHourEpoch(ctx, hourEpoch)
 }
+
+// Add a 10% rate limit for stTIA to Celestia and Osmosis
+func AddStTiaRateLimit(ctx sdk.Context, k ratelimitkeeper.Keeper) error {
+	addRateLimitMsgTemplate := ratelimittypes.MsgAddRateLimit{
```

Comment:
I thought that this was done via proposal which had to be approved via governance... I didn't think we could directly do this via keeper functions

Reply:
Anything is possible in the upgrade handler!